File: src/netcore-api/SIL.XForge.WebApi.Server/Models/Lexicon/LexTaskDashboard.cs
namespace SIL.XForge.WebApi.Server.Models.Lexicon
{
    public class LexTaskDashboard : LexTask
    {
        public LexTaskDashboard()
        {
            Type = Dashboard;
        }

        public int TimeSpanDays { get; set; }
        public int TargetWordCount { get; set; }
    }
}

File: src/Api/Model/Languageforge/Lexicon/Config/LexTaskDashboard.php
<?php

namespace Api\Model\Languageforge\Lexicon\Config;

class LexTaskDashboard extends LexTask
{
    public function __construct()
    {
        $this->type = LexTask::DASHBOARD;
        parent::__construct();
    }

    /**
     * Number of days to view the data
     * @var int
     */
    public $timeSpanDays;

    public $targetWordCount;
}
